### DISCUSSION TITLE
奇忽局战略架构模型及后续讨论

### DISCUSSION THREAD

**Discussion by qihuju** [qihuju/qihuju.github.io#10] [Ideas]
@MRchildNEO 阿呆提供了战略架构模型框架脑图
@Tz-H Tz 将脑图导入了开放协同编辑脑图工具 MindMeister

当前脑图架构（截屏）如下：
![奇忽局：架构战略分析](https://user-images.githubusercontent.com/80709287/113010742-feac4700-9170-11eb-9838-ffc615202f39.png)

脑图协同编辑地址如下：
https://mm.tt/1831602265?t=3evp9XH0Iw

**Comment by Lupingyue** [comment]
@MRchildNEO 前几天跟我和 @MingxuanXie 开了一个小房讨论这个战略架构模型。之后分岔出了几个具体话题，阿呆已经开设[奇忽局的内容架构模型 #9](https://github.com/qihuju/qihuju.github.io/discussions/9)进行专题讨论。
当然这个架构模型内容很多，我们可以继续在这里讨论更多内容。
